### ISSUE TITLE
What does `rolling` use in `plot.py`?

### ISSUE THREAD

**Issue by acgtyrant** [bamos/densenet.pytorch#8]
It seems cut off trainI and occur error in plot.py.

**Comment by bamos** [comment]
It computes the moving average over the training loss/error, and it should be working correctly as long as the window size is larger than the data.
